### PR TITLE
PLATUI-3323 Update play frontend

### PR DESCRIPTION
**Pull request by goodvibes**
https://jira.tools.tax.service.gov.uk/browse/PLATUI-3323

### CODE REVIEW

**Comment by TimothyFothergill** [comment]
At the moment this one is blocked as `hmrc-frontend` upgrade comes first when we bump these. There's a PR open for `hmrc-frontend`: https://github.com/hmrc/hmrc-frontend/pull/407 and then the newest `hmrc-frontend` version will need to be bumped as well.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
...
+### Changed
+
+- Uplifted version of `govuk-frontend` to `v5.7.1`
+- Uplifted version of `hmrcFrontendVersion` to `6.35.0`
+
+### Compatible with
+
+- [hmrc/hmrc-frontend v6.29.0](https://github.com/hmrc/hmrc-frontend/releases/tag/v6.31.0)
+- [alphagov/govuk-frontend v5.5.0](https://github.com/alphagov/govuk-frontend/releases/tag/v5.7.1)
+
```
Comment (oscarduignan) [inline CHANGELOG.md:22]:
```suggestion
### Compatible with

- [hmrc/hmrc-frontend v6.35.0](https://github.com/hmrc/hmrc-frontend/releases/tag/v6.35.0)
- [alphagov/govuk-frontend v5.7.1](https://github.com/alphagov/govuk-frontend/releases/tag/v5.7.1)

```

**Comment by oscarduignan** [comment]
other than that small one it looks good to me, I didn't clock that the coat of arms is a css only change - thought there was a difference in markup too

**Review by oscarduignan — DISMISSED** [review]

**Review by TimothyFothergill — APPROVED** [review]
lgtm

**Pull request merged**